bandwidth/bandwidth-store/src/main/java/com/duckduckgo/bandwidth/store/BandwidthEntity.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.bandwidth.store
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "bandwidth")
+data class BandwidthEntity(
+    @PrimaryKey val id: Int = 0,
```

Comment:
Am I correct the primary key will always be `0` so effectively we'll never have more than one item in the table?
If so I'd probably leave a comment right here in the entity